RadixWallet/Clients/TransactionClient/Models/TransactionModels.swift

```diff
@@ -302,21 +302,14 @@ public struct FeePayerSelectionResult: Equatable, Sendable {
 
 extension ExecutionSummary {
 	func guranteesCost() throws -> RETDecimal {
-		let transaction = try transactionKind()
-		switch transaction {
-		case .nonConforming:
-			return .zero
-		case let .conforming(.general(transaction)):
-			return transaction.accountDeposits.flatMap(\.value).reduce(.zero) { result, resource in
-				switch resource {
-				case .fungible(_, .predicted):
-					result + TransactionFee.PredefinedFeeConstants.fungibleGuaranteeInstructionCost
-				default:
-					result
-				}
+		guard detailedManifestClass == .general else { return .zero() }
```

Comment:
Note that now we are not collapsing all `transfer` kinds in a `general` kind, so here you should have `.general` and `.transfer`

Reply:
ok